src/main/damApi.ts

```diff
@@ -0,0 +1,63 @@
+import fetch from "node-fetch";
+
+const BASE_DKWEBSYS_REQUEST = {
+  modelTypeCode: "2",
+  minseiModelNum: "M1",
+  compId: "1",
+  authKey: "2/Qb9R@8s*",
```

Comment:
🤔

Reply:
amazingly these are common to every win10 dam client